ui/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM node
+FROM ubuntu
```

Comment:
What is the reason for using ubuntu image instead of docker?

Reply:
Hi @JaDogg bro, 
I had problems running with `node` as a base image so I have used `ubuntu` to install nodejs manually and it worked fine.

Reply:
which version of ubuntu this will pick? `FROM ubuntu` is it bionic or xenial. Please check that and if it's xenial then change it to `FROM ubuntu:18.04`

Reply:
I prefer to have to official NodeJs image. This way, when the official repo gets updated, we too get these updates. If you got issues with official repo, we can discuss those.

Reply:
@agentmilindu then let's have a specific version of nodejs base image (let's say 6) so that Bassa angular UI will be compatible. We had a discussion about breakdown due to higher versions of nodejs in this [thread](https://github.com/scorelab/Bassa/issues/375), What would you say? Shall we go this way?

Reply:
@agentmilindu I have fixed it. Please review :)